MyHeartCounts/Onboarding/PreferredWorkoutStep.swift

```diff
@@ -0,0 +1,114 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SFSafeSymbols
+import SpeziOnboarding
+import SpeziViews
+import SwiftUI
+
+
+struct PreferredWorkoutStep: View {
+    struct NotificationTime: Hashable, Codable, Sendable {
+        var hour: Int
+        var minute: Int = 0
+    }
+    
+    @Environment(\.calendar)
+    private var calendar
+    @Environment(ManagedNavigationStack.Path.self)
+    private var path
+    
+    @State private var viewState: ViewState = .idle
+    @State private var workoutType: WorkoutType?
+    @State private var notificationTime = NotificationTime(hour: 9)
+    
+    var body: some View {
+        OnboardingPage(title: "Workout Preference", description: "WORKOUT_PREFERENCE_TEXT") {
+            selectionForm
+            DatePicker(
+                "Notification Time",
+                selection: Binding<Date> {
+                    calendar.date(
+                        bySettingHour: notificationTime.hour,
+                        minute: notificationTime.minute,
```

Comment:
Tagging @PaulGoldschmidt here: I think we should probably only allow full hour increments here as we only schedule the function every hour? Or should we schedule it every minute/an other interval? We would need to sync this here.